aptos-move/aptos-vm/src/move_vm_ext/write_op_converter.rs

```diff
@@ -65,6 +67,118 @@ impl<'r> WriteOpConverter<'r> {
         }
     }
 
+    pub(crate) fn convert_resource_group_v1(
+        &self,
+        state_key: &StateKey,
+        group_changes: AccountChangeSet,
+        legacy_creation_as_modification: bool,
```

Comment:
I don't think we need this for anything new

Reply:
ah, because this only gets called with new version, so doesn't need to worry about backwards compatibility, cool!